avalon/nuke/pipeline.py

```diff
@@ -105,20 +171,18 @@ def parse_container(node):
         dict: The container schema data for this container node.
 
     """
-    data = lib.read(node)
-
-    # (TODO) Remove key validation when `ls` has re-implemented.
-    #
-    # If not all required data return the empty container
-    required = ["schema", "id", "name",
-                "namespace", "loader", "representation"]
-    if not all(key in data for key in required):
-        return
+    data = lib.get_avalon_knob_data(node)
 
     # Store the node's name
     data["objectName"] = node["name"].value()
     # Store reference to the node object
     data["_node"] = node
+    # Get containerized nodes
+    if node.fullName() == "%s.%s" % (AVALON_CONTAINERS, node.name()):
+        data["_members"] = lib.lsattr("avalon:containerId",
+                                      value=data["avalonId"],
+                                      group=nuke.toNode(AVALON_CONTAINERS),
+                                      recursive=True)
```

Comment:
@tokejepsen 
> I think the backdrop workflow within the `AVALON_CONTAINERS` group could use some documentation and code. For example as a developer how do I get the nodes for a container in the group?

Yeah, you are right, that seems very implicit. Currently it looks like you can only get them from the entry `"_member"` in the parsed container.

Can't remember why I implement it like this. :/
Any suggestion ?

Reply:
Hmm, long time since I posted this, but I think generally I found it confusing about how to fetch nodes to pass to `sync_copies`.
Ideally we would work with similar arguments to get the nodes in the container group and what to pass to `sync_copies`.

Reply:
Not sure if this helps, currently I only use it inside our loader plugin's update method, like this

https://github.com/MoonShineVFX/reveries-config/blob/a5c56c5d3ee6e87498b024004a41e0b80fc48501/plugins/nuke/load/load_renderlayer.py#L244-L257

Haven't had the needs to find backdrop(container) nodes in other cases.
Maybe we could decouple this
```python
# Get container backdrop child nodes
lib.lsattr("avalon:containerId",
           value=data["avalonId"],  # container's id
           group=nuke.toNode(AVALON_CONTAINERS),
           recursive=True)
```
into another function and expose it ?

Reply:
Yeah, sounds like a good idea.